apps/web/src/components/layouts/sidebar.tsx

```diff
@@ -1,25 +1,32 @@
 import { Logo } from "@/components/ui/logo";
+import { useResponsive } from "@/providers/responsive-provider";
 
 export const Sidebar = () => {
+    const { isNavsOpen } = useResponsive();
+
     return (
-        <nav className="scrollbar-none scrollbar-hide fixed h-full w-20 overflow-y-auto bg-zinc-900">
-            {/* Dashboard */}
-            <div className="flex items-center justify-center border-gray-600 border-b py-2">
-                <Logo alt="Dashboard" />
-            </div>
-            {/* Servers */}
-            <div className="flex flex-col items-center justify-center gap-4 pt-4">
-                {Array.from({ length: 20 }).map((_, index) => (
-                    <div
-                        key={index}
-                        className="group relative flex w-full items-center justify-center"
-                    >
-                        <Logo alt="server" />
-                        {/* Notifcation */}
-                        <div className="absolute top-1/3 left-0 flex h-2 w-1 items-center justify-center overflow-hidden rounded-full bg-white duration-100 group-hover:top-3 group-hover:h-6"></div>
-                    </div>
-                ))}
-            </div>
-        </nav>
+        isNavsOpen && (
```

Comment:
- use consitional rendering instead of `&&` bcs of possible bugs.
- also `!!isNavsOpen` is good

Reply:
`&&` is a new way to conditionally render a component there is no problem with it.

Reply:
@GoatMoaz 
Check this video:
https://youtu.be/Rr5AqASIyxw?si=XTgQHPuSSZRUNpqE
![Screenshot_20241001_173933_YouTube](https://github.com/user-attachments/assets/022fa67a-3344-42d4-b6bd-c4e2646679e8)

Reply:
It's not a problem only if the value is always true or false.

Reply:
good the `isNavsOpen` state is always true or false even initially I set it to true not undefined.